bioimageio/spec/model/v0_4/schema.py

```diff
@@ -527,7 +531,7 @@ def get_min_shape(t) -> numpy.ndarray:
         "(#https://en.wikipedia.org/wiki/ISO_8601) format.",
     )
 
-    training_data = fields.Union([fields.String(bioimageio_description="dataset id"), fields.Nested(Dataset())])
+    training_data = fields.Union([fields.Nested(Dataset()), fields.Nested(LinkedDataset())])
```

Comment:
@oeway @constantinpape here we check first for an in-place dataset rdf and otherwise for the LinkedDataset that currently only consists of `id`. unknown fields are not allowed for the `LinkedDataset`